### ISSUE TITLE
Detect if payload is stringified too many times

### ISSUE THREAD

**Issue by jmandel** [microsoft/health-cards-validation-SDK#50]
From an example shared on github:

```
$ node . -t qr  -p ~/Downloads/Screen-Shot-2021-04-13-at-4.06.39-PM.png 
SMART Health Card Validation SDK v0.4.3
balancedSizeBuffer: 5
/~/health-cards-validation-SDK/js/src/jws-compact.js:159
                    if (payload.iss.slice(0, 8) !== 'https://') {
                                    ^
```

The payload here was like `JSON.stringify(JSON.stringify({iss: "https://example"}))` -- so JSON.parse succeeds, but the result is a string instead of an object. It'd be good to detect this, or even if there's no special logic for this case, it'd be good to fail more gracefully when `payload.iss` is undefined.

**Comment by christianpaquin** [comment]
Probably isn't unique to this field. We can see how to do this throughout the SDK.

**Comment by christianpaquin** [comment]
Do you still have the problematic QR code, @jmandel, and if so, can you share it?

**Comment by jmandel** [comment]
https://chat.fhir.org/#narrow/stream/284830-smart.2Fhealth-cards/topic/SHC.20Pre-event.20Discussion/near/234393066

![image](https://user-images.githubusercontent.com/313089/115793122-333aa980-a391-11eb-9976-2f22eb3d85d6.png)

**Comment by christianpaquin** [comment]
Fixed by #71
